Mathlib/RingTheory/Valuation/Basic.lean

```diff
@@ -398,10 +420,26 @@ theorem comap {S : Type*} [Ring S] (f : S →+* R) (h : v₁.IsEquiv v₂) :
     (v₁.comap f).IsEquiv (v₂.comap f) := fun r s => h (f r) (f s)
 #align valuation.is_equiv.comap Valuation.IsEquiv.comap
 
+/-- Restate the definition of `Valuation.IsEquiv` into a theorem. -/
+theorem val_le (h : v₁.IsEquiv v₂) {r s : R} : v₁ r ≤ v₁ s ↔ v₂ r ≤ v₂ s :=
+  h r s
+
+theorem val_lt (h : v₁.IsEquiv v₂) {r s : R} : v₁ r < v₁ s ↔ v₂ r < v₂ s := by
+  simpa only [not_le] using (h s r).not
+
 theorem val_eq (h : v₁.IsEquiv v₂) {r s : R} : v₁ r = v₁ s ↔ v₂ r = v₂ s := by
   simpa only [le_antisymm_iff] using and_congr (h r s) (h s r)
 #align valuation.is_equiv.val_eq Valuation.IsEquiv.val_eq
 
+theorem val_le_one (h : v₁.IsEquiv v₂) {r : R} : v₁ r ≤ 1 ↔ v₂ r ≤ 1 := by
```

Comment:
Can you explain why these three results are useful?

Reply:
Same as above.